corgidrp/mocks.py

```diff
@@ -88,7 +129,7 @@ def create_default_headers():
     exthdr['DPAM_H'] = 1.0
     exthdr['DPAM_V'] = 1.0
     exthdr['DATETIME'] = '2024-01-01T11:00:00.000Z'
-    exthdr['DATA_LEVEL'] = "L1"
+    exthdr['DATA_LVL'] = "L1"
```

Comment:
The current data specification is using DATA_LEVEL, unless you know this has changed.

Reply:
Right, I only changed this because having so many letters threw a warning that it makes it a HIERARCH-type keyword, which seemed unnecessary. Alex just told me though that this was requested specifically from the PS team, so I don't think we should change it for now. Ultimately I think the functionality and use will be the same as a normal keyword, but for visually inspecting headers it's a slight pain. We could consider changing it later. @mygouf are you aware of a reason that it was chosen to be this way?